src/app/token-option-field-component-factories/spend-for-quiz-revision-token-option-field-component-factory.ts

```diff
@@ -38,7 +38,7 @@ export class SpendForQuizRevisionTokenOptionFieldComponentFactory extends TokenO
     ] {
         const assignmentField = createFieldComponentWithLabel(
             StringInputFieldComponent,
-            'Assignment Name',
+            'Canvas Assignment whose Available Until should change',
```

Comment:
Please see comment on line 62 of `spend-for-assignment-resubmission-token-option-field-component-factory.ts`.